apps/atomy-q/API/database/factories/AwardFactory.php

```
@@ -0,0 +1,76 @@
+    public function signedOff(): static
+    {
+        return $this->state(fn (array $attributes) => [
+            'status' => 'signed_off',
+            'signoff_at' => now(),
+            'signed_off_by' => User::factory(),
+        ]);
+    }
+
+    public function protested(): static
+    {
+        return $this->state(fn (array $attributes) => [
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)